multimodal/tests/unittests/test_text_recognition.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import pytest
+import requests
+from mim.commands.download import download
+from mmocr.utils.ocr import MMOCR
+from PIL import Image
+
+from autogluon.multimodal import MultiModalPredictor
+
+
+def download_sample_images():
+    url = "https://raw.githubusercontent.com/open-mmlab/mmocr/main/demo/demo_text_recog.jpg"
+    image = Image.open(requests.get(url, stream=True).raw)
+    mmocr_image_name = "demo.jpg"
+    image.save(mmocr_image_name)
+
+    return mmocr_image_name
+
+
+# TODO: when using crnn checkpoint, the results are wrong.
```

Comment:
Why this is the case? Is it just this model or multiple models have wrong results?

Reply:
Just this model. The value of the 'score' item do not match.
<img width="573" alt="image" src="https://user-images.githubusercontent.com/54358503/192126769-4d053506-c7f0-4e79-bd0f-48e952231b25.png">

Reply:
I have tried all the models that support text recognition. And they can generate correct results except this model.

Reply:
Cool, thanks. Then we can dive into this issue in later PRs.